kafka-rest/src/test/java/io/confluent/kafkarest/unit/ConsumerManagerTest.java

```diff
@@ -15,6 +15,7 @@
  **/
 package io.confluent.kafkarest.unit;
 
+import io.confluent.kafkarest.*;
```

Comment:
nit: we usually prefer explicit imports

Reply:
yeah sorry, I haven't been able to set up IDEA to follow this